nomad/keyring_endpoint.go

```diff
@@ -348,6 +348,15 @@ func (k *Keyring) Delete(args *structs.KeyringDeleteRootKeyRequest, reply *struc
 		return fmt.Errorf("active root key cannot be deleted - call rotate first")
 	}
 
+	// make sure the key was used to encrypt an existing variable
+	rootKeyInUse, err := snap.IsRootKeyInUse(args.KeyID)
+	if err != nil {
+		return err
+	}
+	if rootKeyInUse && !args.Force {
+		return fmt.Errorf("root key in use, cannot delete")
```

Comment:
There is no formatting, so we could use the below (the suggestion won't compile as the file also needs the new import):
```suggestion
		return errors.New("root key in use, cannot delete")
```

Reply:
yeah force of habit I guess, but `errors.New` is a better choice here